packages/@divvi/mobile/src/alert/AlertBanner.test.tsx

```diff
@@ -48,8 +48,8 @@ describe('AlertBanner', () => {
       )
       const banner = getByTestId('infoBanner')
 
-      expect(banner).toHaveTextContent('Declaration')
-      expect(banner).toHaveTextContent('This is your shadow speaking')
+      expect(banner).toHaveTextContent('Declaration', { exact: false })
```

Comment:
Allows us to use the existing pattern of testing for text; otherwise, it would be the entire translation string that does not match what the user sees.

Reply:
@jeanregisser We have to specify false; otherwise, it'll match the whole translation string. Setting the exact match to false seemed to be the better, as it tests what the user would see.

`swapScreen.transactionDetails.exchangeRate1 cUSD ≈ 1.23456 CELO` v.s. `1 cUSD ≈ 1.23456 CELO`